src/app/services/technical-record/technical-record.service.ts

```diff
@@ -304,6 +304,8 @@ export class TechnicalRecordService {
   ) {
     const url = `${environment.VTM_API_URI}/vehicles/documents/plate`;
 
+    const emailExists = techRecord?.applicantDetails?.emailAddress !== undefined && techRecord?.applicantDetails?.emailAddress !== '';
```

Comment:
you could remove this and use an ||  instead of ??. this would force a truthy check which is false for undefined and empty string